modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -12571,7 +12571,7 @@ export const console: ConsoleNS = {
                 title: "Users"
             },
             usersEdit: {
-                backButton: "Go back to Users",
+                backButton: "Go back to {{type}}",
```

Comment:
ned to fix this for french and sinhala as well.

Reply:
Added with [a1e3a33](https://github.com/wso2/identity-apps/pull/4669/commits/a1e3a335219bb95a418df891843749cb2ed944cc)